src/bootstrap/native.rs

```diff
@@ -239,7 +239,7 @@ impl Step for Llvm {
 
         builder.update_submodule(&Path::new("src").join("llvm-project"));
         if builder.llvm_link_shared()
-            && (target.contains("windows") || target.contains("apple-darwin"))
+            && (target.contains("windows") || (target.contains("apple-darwin") && !cfg!(unix)))
```

Comment:
`target.contains("apple-darwin") && !cfg!(unix)))`

When will both of these things be true? Shouldn't we just remove apple-darwin?